hw01/src/main/scala/ru/otus/scala/call_count/dao/CallCountDaoImpl.scala

```diff
@@ -0,0 +1,11 @@
+package ru.otus.scala.call_count.dao
+
+class CallCountDaoImpl extends CallCountDao {
+  var counter = 0L
+
+  def get(): Long = {
+    counter
+  }
+
+  def increment(): Unit = counter = counter + 1
```

Comment:
Я бы предпочёл оставлять фигурные скобочки если мы работаем с "не чистым" ФП кодом.
```suggestion
  def increment(): Unit = {
    counter = counter + 1
  }
```

Причина в том что некоторые выражения без скобочек тоже скомпилируются:

```
  def increment(): Unit =
    counter = counter + 1
    counter = counter + 1      // представим что нам зачем-то это нужно
```

Кто-то может ожидать что код сверху инкрементирует значение на 2, но по факту второе выражение будет вызвано во время создания объекта. С фигурными скобочками намерение автора будет чуть более явным.

С чистыми функциями такая проблема возникает реже, т.к. чаще всего происходит ошибка на этапе компиляции из-за несоответствия типов.